modular_skyrat/modules/alerts/code/security_level_datums.dm

```diff
@@ -68,3 +68,5 @@
 	elevating_to_configuration_key = /datum/config_entry/string/alert_gamma
 	shuttle_call_time_mod = 0.25
 	sound = 'modular_skyrat/modules/alerts/sound/security_levels/gamma_alert.ogg'
+	looping_sound = 'modular_skyrat/modules/alerts/sound/misc/alarm_delta.ogg'
+	looping_sound_interval = 8 SECONDS
```

Comment:
Why would we play the delta alarm when we already have the gamma alert sound? If you want the gamma alert to be looping, just make it looping, don't use the delta one as a "placeholder" that'd probably overlap with the regular `gamma_alert.ogg` one (which is by far my favorite alert sound)

Reply:
these short eerie chimes that play once every 8 seconds on delta sounded fitting for me on gamma, and they helped keep up the tension long after the gamma alert sound expires (which is what happened to me during armageddon CMEs), I can remove it if it really doesn't fit the aesthetic of gamma though

for reference:
https://user-images.githubusercontent.com/47710522/188508721-8901b78f-d40f-4f95-a080-1e5a51f27541.mp4

Reply:
I still think we should keep it to the gamma one instead, and make sure that it loops, because I still think it's one of the most immersive alert sounds we have (I still get the chills every time it plays).

Reply:
do we have a gamma looping sound?